lib/node_modules/@stdlib/math/base/special/bessely0/benchmark/python/benchmark.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""Benchmark bessely0."""
```

Comment:
Similarly, for SciPy benchmarks, we are a bit more specific in our description.

> Benchmark.special.bessely0.

This is so a person opening the file can see more immediately what is being tested without have to dig around in the benchmark code.

Reply:
All good, but I think that should be:

> `Benchmark scipy.special.y0.`

Reply:
Yes, sorry. You are correct.